scripts/which_clang_files

```diff
@@ -0,0 +1,33 @@
+#! /bin/sh
```

Comment:
Love this! Handles both cases! I'm also glad you're the writing bash and not me. Matthew is our resident bash expert so please have him review this.

Reply:
I wouldn't say I'm doing a good job at writing the bash :)
Will certainly have Matthew take a look before landing